test_integration/02_stake_and_mint/02_stake_and_mint_proof.js

```diff
@@ -0,0 +1,330 @@
+// Copyright 2019 OpenST Ltd.
```

Comment:
Can you fix linting issues of this file.

Reply:
Fixed.